src/table_builder.cc

```diff
@@ -12,7 +12,31 @@ void TitanTableBuilder::Add(const Slice& key, const Slice& value) {
     return;
   }
 
-  if (ikey.type != kTypeValue || value.size() < cf_options_.min_blob_size) {
+  std::shared_ptr<VersionSet> vset;
+  if (ikey.type != kTypeValue &&
+      cf_options_.blob_run_mode == TitanBlobRunMode::kFallback &&
+      (vset = vset_ref_.lock()) != nullptr) {
+    Slice copy = value;
+    uint64_t cf_id = GetTableProperties().column_family_id;
```

Comment:
getting CF id from table property looks hacky to me. Can we pass CF id to table factory on `TitanDBImpl::Open()` then pass the id to table builder from table factory?

Reply:
Builder holds an id data, I missed it.